### PR TITLE
Add security policy publication to the checklist

### PR DESCRIPTION
**Pull request by MarcinHoppe**
This should be merged after openjs-foundation/cross-project-council#633 lands.

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by MarcinHoppe** [comment]
Now that https://github.com/openjs-foundation/cross-project-council/pull/633 landed, this PR should be merged, too.

**Review by jorydotcom — APPROVED** [review]

**Review by tobie — COMMENTED** [review]
Isn't this only for Growth and Impact project, though? If so should we call it out?

**Comment by mhdawson** [comment]
@tobie I don't think so. I'd thought we'd agreed that its a minimal enough requirement to apply to all

**Comment by tobie** [comment]
Can we merge this now?

**Comment by mhdawson** [comment]
@tobie I think so but there is a conflict now. @MarcinHoppe any chance you can resolve the conflict and then I'll land ?

**Comment by MarcinHoppe** [comment]
@mhdawson I resolved the conflict, I think it is ready to land.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-# project-onboarding
-Public repo to manage the process of onboarding projects
+# project-status
+Public repo to manage the process of onboarding new projects
+and helping Growth stage projects become Impact projects.
```
Comment (mhdawson) [inline README.md:3]:
```suggestion
# project-onboarding
Public repo to manage the process of onboarding projects
```

**Pull request merged**